src/app/shared/learning-object-ratings/learning-object-ratings.component.ts

```diff
@@ -38,6 +43,13 @@ export class LearningObjectRatingsComponent implements OnInit {
     return false;
   }
 
+  isResponseAuthor(index: number): boolean {
+    if (this.ratings[index].response[0].user.username === this.auth.username) {
```

Comment:
This could be a one-liner instead: `return this.ratings[index].response[0].user.username === this.auth.username`

Reply:
Is there a reason this was resolved without changing the code?